.github/workflows/test.yml

```
@@ -21,3 +21,22 @@ jobs:
 
       - name: Test packages
         run: yarn test
+
+  e2e-tests:
+    name: Run e2e tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2.1.4
+        with:
+          node-version: 14
+
+      - run: |
+          yarn install --frozen-lockfile
```

Could we name steps, so they're better visible on UI (like [here](https://github.com/onfido/castor/runs/2321776409?check_suite_focus=true)):

```suggestion
      - name: Checkout Git repository
        uses: actions/checkout@v2

      - name: Setup Node
        uses: actions/setup-node@v2.1.4
        with:
          node-version: 14

      - name: Install Node dependencies
        run: yarn install --frozen-lockfile

      - name: Test packages
        run: |
```

---

I find it easy to understand `Run actions/setup-node` for example, but don't mind naming the steps.

---

```suggestion
      - name: Checkout Git repository
        uses: actions/checkout@v2
        
      - name: Setup Node
        uses: actions/setup-node@v2.1.4
        with:
          node-version: 14
          
      - name: Install Node dependencies
        run: yarn install --frozen-lockfile
        
      - name: Run Cypress tests
        run: |
```

---

Just to match with other actions.

Yet again, I don't mind then not using names, but it should be one PR removing naming from all actions rather than some actions having names, some not...

---

Renamed the last one slightly compared to your suggestion.